### PR TITLE
Make changes

### PR DESCRIPTION
**Pull request by telinc1**
The following changes were made:

- Foo
- Bar
- Baz

### CODE REVIEW

File: friends.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@
 - bugfix
 - see all friends
 - getMyFriends
+- add new functionality
```
Comment (NaskoIT) [inline friends.txt:6]: Teo, I think that here you can use beter naming

File: new_file.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+new file
```
Comment (NaskoIT) [inline new_file.txt:1]: I think that you can use this design pattern:

**Review by NaskoIT — CHANGES_REQUESTED** [review]

File: friends.txt (hunk 1 above)
Comment (telinc1) [reply friends.txt:6]: The names are fine

File: new_file.txt (hunk 2 above)
Comment (telinc1) [reply new_file.txt:1]: I will fix it

**Review by NaskoIT — APPROVED** [review]

**Pull request merged**